src/components/Anchor/Anchor.css

```diff
@@ -0,0 +1,8 @@
+.anchor {
+  @apply text-text-default font-semibold;
+  text-decoration: underline;
```

Comment:
Can we use the [tailwind class](https://v2.tailwindcss.com/docs/text-decoration) here?

Reply:
I **think** the goal is to avoid writing any CSS in these files and instead try to use only the Tailwind @apply classes.

Reply:
Fixed